### DISCUSSION TITLE
百度网盘的视频，网页里能播放，转到potplayer里就不能了

### DISCUSSION THREAD

**Discussion by gwtycl** [AlistGo/alist#4723] [Q&A]
是超级会议，也改了ua，网页里放4K秒开无卡顿，可是转到potplayer里就无法播放了（阿里云盘没问题，都能播放），是因为改ua的那个插件的问题吗？该如何解决呢？

**Comment by anwen-anyi** [comment]
你网页改了UA  ， potplayer又没改肯定是无法播放了吧？

**Reply by gwtycl** [reply]
网页是插件改的，potplayer里面咋改啊？
